Update 3rd-party plugins to NIM compatible versions

f1ames opened the pull request:
This PR updates MathType and WProofreader to latest versions (NIM compatible).

I also fixed license key message as it mentioned `index.ts` instead of `index.js`.

Fixes #41.

pszczesniak reviewed (approved):
LGTM :+1:

Witoso commented:
Related to this #43 

In some older snippets, I saw a comment above this plugin with it being commented out. Maybe let's do this?

f1ames commented:
> Related to this #43 
> 
> In some older snippets, I saw a comment above this plugin with it being commented out. Maybe let's do this?

@Witoso You mean to comment out WProofreader plugin by default? Or addin explanation on "blocked" localhost? For commenting out this plugin, it's "WProofreader" so I would assume it should be there, enabled by default 🤔

f1ames commented:
Ok, I have added comment to each demo with WProofreader to address #43. Also fixed some demos wich weren't updated before.

@pszczesniak could you take another look if the message makes sense?

Witoso commented on feature-rich/index.js:
```diff
@@ -22,6 +22,9 @@ const CKBOX_TOKEN_URL = '';
 // For more info how to get the key, see https://ckeditor.com/docs/ckeditor5/latest/features/spelling-and-grammar-checking.html.
+//
+// IMPORTANT: WProofreader has a mechanism preventing it to run on 'localhost' domain. To test it locally, you need to use
+// a different domain name, by, for example, mapping 'localhost' to 'fake.ckeditor.com' in your '/etc/hosts' file.
```
```suggestion
// a different domain name, by, for example, mapping 'localhost' to 'localhost.dev' in your '/etc/hosts' file.
```

I would rather not leak fake.ckeditor.com.

pszczesniak replied:
`.dev` domains can be locally? AFAIR when Google bought this makes some redirects even locally on Chrome? :thinking:

Maybe `dev.your-domain.com`?

Witoso replied:
ah, you're right, I saw also `dev.localhost` pattern I think.

f1ames replied:
Ok, I'll fix it 👍

I'm just confirming now how it works exactly. I see with trial key, all domains are allowed by default 🤔 Probably, that's not the case for paid plan. I'm checking this and will adjust the message if needed.

f1ames replied:
Ok, I talked to @Stalq, and it seems this is no longer relevant. It worked like that some time ago but now pricing model is based on the number of spellchecked words (AFAIU).

There is a way to change allowed/excluded domains on WSC side to e.g. to exclude some, but not automatically (requires contacting their support) so this is a really custom thing.

I'll restore the original comments. Sorry for not checking this in more details earlier 😓 🙈

f1ames commented:
Merging now, since the only requested change is no longer relevant (due to https://github.com/ckeditor/ckeditor5-demos/pull/42#discussion_r1656960506).
